sql/core/src/main/scala/org/apache/spark/sql/execution/ui/SQLAppStatusListener.scala

```diff
@@ -321,6 +346,11 @@ class SQLAppStatusListener(
     }
   }
 
+  private def removeLiveEntities(exec: LiveExecutionData): Unit = {
```

Comment:
`removeLiveExecution`. But really this is only called from a single place, so just inline it?

Reply:
Done.